third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -227,6 +227,7 @@ function log(...args) {
   log_(args.join(' '));
 }
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
left these two utility functions in because they're tiny and more likely to be useful generally for other changes we might make to this script.